roles/kubernetes-apps/metallb/defaults/main.yml

```diff
@@ -1,5 +1,7 @@
 ---
 metallb_enabled: false
+metallb_speaker_image_repo: "{{ docker_image_repo }}/metallb/speaker"
+metallb_controller_image_repo: "{{ docker_image_repo }}/metallb/controller"
```

Comment:
All repo definitions are written on roles/download/defaults/main.yml like
```
dashboard_metrics_scraper_repo: "{{ docker_image_repo }}/kubernetesui/metrics-scraper"
```
at this time. Why don't we write like the existing ones instead of roles/kubernetes-apps/metallb/defaults/main.yml ?

Reply:
I've moved these variables to the download role's defaults

Reply:
Thanks for doing that, I confirmed it.